be/src/product/dto/update-product.input.ts

```diff
@@ -1,8 +1,7 @@
-import { CreateProductInput } from './create-product.input';
-import { InputType, Field, Int, PartialType } from '@nestjs/graphql';
+import { InputType, Field, Int } from '@nestjs/graphql';
 
 @InputType()
-export class UpdateProductInput extends PartialType(CreateProductInput) {
```

Comment:
Not like this :D

Keep the extension just delete the fields. Read the doc on what the  `PartialType` does and why to use it.